src/components/Sum/Sum.jsx

```diff
@@ -1 +1,15 @@
-// export const Sum = () => ();
+export const Sum = ({ a = 0, b = 0 }) => (
+  <p>
+    Sum of
+    {' '}
+    {a}
+    {' '}
+    and
+    {' '}
+    {b}
+    {' '}
+    is
+    {' '}
+    {a + b}
+  </p>
```

Comment:
You could simply write:
```suggestion
  <p>
    {`Sum of ${a} and ${b} is ${a + b}`}
  </p>
```